src/app/components/common/govuk-heading/govuk-heading.component.html

```diff
@@ -1,21 +1,21 @@
 @switch (tag) {
   @case ('h1') {
-    <h1 [class]="headingClass">
+    <h1 [class]="headingClass" tabindex="0">
       <ng-container *ngTemplateOutlet="text"></ng-container>
     </h1>
   }
   @case ('h2') {
-    <h2 [class]="headingClass">
+    <h2 [class]="headingClass" tabindex="0">
       <ng-container *ngTemplateOutlet="text"></ng-container>
     </h2>
   }
   @case ('h3') {
-    <h3 [class]="headingClass">
+    <h3 [class]="headingClass" tabindex="0">
       <ng-container *ngTemplateOutlet="text"></ng-container>
     </h3>
   }
   @case ('h4') {
-    <h4 [class]="headingClass">
+    <h4 [class]="headingClass" tabindex="0">
```

Comment:
Perhaps tabindex could be passed in / configurable by the parent?/

Reply:
As mentioned above, tabindex should almost always be 0